analyzer-service/calculations.py

```diff
@@ -4,20 +4,35 @@
 import statsmodels.api as sm
 from datetime import datetime
 
+import time
```

Comment:
Once we feel good about the stability of our calculations process we might think about removing these timing statements but for right now I like using them as a good way of profiling/benchmarking our code

Reply:
Yep, I'm on board.